test/decl/protocol/protocol_with_superclass.swift

```diff
@@ -351,3 +351,11 @@ func usesProtoRefinesClass2<T : ProtoRefinesClassComposition>(_ t: T) {
   t.genericMethod((1, 2))
   let _: BaseProto = t
 }
+
+class SR10483_C: SR10483_P {}
```

Comment:
Actually, let us rename these to "issueNNNN", and I will add the links too.

Reply:
Or some more neutral names would work too if you can come up with good ones. I usually avoid suggesting renaming in tests since I often struggle with that myself :)

Reply:
"issueNNNN" is IMHO the best one I could come up with as of now. I think "issue" is pretty neutral, in the sense that it need not apply just to bugs, and perhaps better than an abbreviation that is often hard to decipher and more strongly tied to a specific tracking system (like, I still wonder what SR stands for, Swift Radar? 😅).

@hborla What do *you* think?

Reply:
> Yeah I like this strategy too! Lately I've been using it but with more descriptive names (e.g. what the actual issue is) instead of an issue/jira/radar number. An issue link in a comment is perfectly sufficient if the issue has helpful context.

Me and Pavel misunderstood each other—I was referring to the "naming convention" that is to replace "SRXXX" , which I proposed to be "issueYYY". But I totally concur that wrapping issue-specific test cases in functions/blocks when possible and mentioning the issue once is a lot neater that sticking the ID in every identifier.

Reply:
I did understand what you meant and `issueXXX` is fine, just like we do for `rdarYYY` and I should have clarified that my suggestion is separate, sorry!